wp-content/themes/wp-starter/plopfile.js

```diff
@@ -18,6 +18,7 @@ module.exports = function (plop) {
 				type: 'input',
 				name: 'themeslug',
 				message: 'What is the slug for your theme?',
+				default: 'wp-starter',
```

Comment:
We just need to make sure in the `bin/composer-scripts` we're including this file in string replacements for `wp-starter`. I want to say I saw one other place we need to update, but can't remember at the moment.

Reply:
Gotcha I can check that.